examples/pytorch_ignite/gin.py

```diff
@@ -0,0 +1,214 @@
+        print()
+
+    # create an handler to save checkpoints of the
+    # model based on Accuracy on the training set
+    checkpoint_handler = ignite.handlers.Checkpoint(
+        {'model': model}, log_dir, n_saved=2,
+        score_name=list(metrics.keys())[0],
+        filename_pattern="best_trn_epoch-{global_step}" +
+        "-{score_name}-{score}.pt",
+        global_step_transform=ignite.handlers.global_step_from_engine(trainer))
+    trn_evaluator.add_event_handler(ignite.engine.Events.EPOCH_COMPLETED,
+                                    checkpoint_handler)
+
+    # create an handler to save checkpoints of the
+    # model based on Accuracy on the validation set
+    checkpoint_handler = ignite.handlers.Checkpoint(
+        {'model': model}, log_dir, n_saved=2,
+        score_name=list(metrics.keys())[0],
+        filename_pattern="best_val_epoch-{global_step}" +
+        "-{score_name}-{score}.pt",
+        global_step_transform=ignite.handlers.global_step_from_engine(trainer))
+    val_evaluator.add_event_handler(ignite.engine.Events.EPOCH_COMPLETED,
+                                    checkpoint_handler)
+
+    # create a tensorboard logger to write logs
+    tb_logger = ignite.contrib.handlers.tensorboard_logger.TensorboardLogger(
+        log_dir=osp.join(log_dir, "tb_logs"))
+    # Attach the logger to the trainer to log training loss at each iteration
+    tb_logger.attach_output_handler(
+        trainer, event_name=ignite.engine.Events.ITERATION_COMPLETED,
+        tag="training", output_transform=lambda loss: {"loss_iteration": loss})
+    # Attach the logger to the trainer to log training loss at each epoch
+    tb_logger.attach_output_handler(
+        trainer, event_name=ignite.engine.Events.EPOCH_COMPLETED,
+        tag="training", output_transform=lambda loss: {"loss_epoch": loss})
+    # Attach the logger to the train evaluator
+    # to log training metrics at each epoch
+    tb_logger.attach_output_handler(
+        trn_evaluator,
+        event_name=ignite.engine.Events.EPOCH_COMPLETED,
+        tag="training",
+        metric_names='all',
+        global_step_transform=ignite.handlers.global_step_from_engine(trainer),
+    )
+    # Attach the logger to the validation evaluator
+    # to log validation metrics at each epoch
+    tb_logger.attach_output_handler(
+        val_evaluator,
+        event_name=ignite.engine.Events.EPOCH_COMPLETED,
+        tag="validation",
+        metric_names='all',
+        global_step_transform=ignite.handlers.global_step_from_engine(trainer),
+    )
+    # Attach the logger to the test evaluator to log test metrics at each epoch
+    tb_logger.attach_output_handler(
+        test_evaluator,
+        event_name=ignite.engine.Events.EPOCH_COMPLETED,
+        tag="test",
+        metric_names='all',
+        global_step_transform=ignite.handlers.global_step_from_engine(trainer),
+    )
+    tb_logger.close()
+
+    # run the trainer
+    trainer.run(loader_trn, max_epochs=50)
```

Comment:
The perfomance of the model are pretty bad. Don't know if this is normal, being just an example (I didn't run the original example with pytorch lighting).

Reply:
What does that mean? What's the final performance of the model?

Reply:
For the training set like 0.80 - 0.85 accuracy, starting at epoch 0 from 0.6-0.7. For the validation and test the accuracy does not increase during training